src_test/ppx_deriving/error.t

```diff
@@ -0,0 +1,85 @@
+  $ cat >dune-project <<EOF
+  > (lang dune 3.0)
+  > EOF
+  $ cat >dune <<EOF
+  > (library
+  >  (name test)
+  >  (preprocess
+  >   (staged_pps ppx_import)))
+  > EOF
+
+Functor error
+  $ cat >test.ml <<EOF
+  > [%%import: type t =  Map.Make(String).t]
+  > EOF
+
+  $ dune build 2>&1 | sed 's/\(Command line: \).*/\1Error/'
+  File ".test.objs/byte/_unknown_", line 1, characters 0-0:
+  Fatal error: exception Invalid_argument("Ppxlib.Longident.flatten")
+  File "test.ml", line 1:
+  Error: Error while running external preprocessor
+  Command line: Error
```

Comment:
The full error looks like

```
Command line: .ppx/730a1ffc71e71794f0005821abd2de02/ppx.exe --as-ppx --cookie 'library-name="test"' '/var/folders/8z/fhgbsxjn4nj6hbnrh7mx7wr80000gn/T/build_5d43fc_dune/build_d0b97e_dune/camlppx1b1f2c' '/var/folders/8z/fhgbsxjn4nj6hbnrh7mx7wr80000gn/T/build_5d43fc_dune/build_d0b97e_dune/camlppxadc1c2'
```

But some parts change upon every compilation, hence why I get rid of that part.